static/app/components/dropdownMenuItemV2.tsx

```diff
@@ -209,6 +209,7 @@ const MenuItem = withRouter(
         as={renderAs}
         isDisabled={isDisabled}
         data-test-id={item.key}
+        {...(item.to && {'data-test-href': item.to})}
```

Comment:
I think we should not introduce one more` data-test` attribute ... shouldn't we only check the data-test-id instead?

Reply:
Added this to pass the `href` tests in `issueWidgetCard.spec`, [like this one](https://github.com/getsentry/sentry/blob/vuluong/dashboard-card-dropdown/tests/js/spec/views/dashboardsV2/issueWidgetCard.spec.tsx#L131-L134).

I tried to add `a` tags with normal `href` attributes that we could test for inside `DropdownMenuItem`, but couldn't do it given the way `react-aria` handles click/tap events (the links would just not work).

Another option is to only check that these menu items exist, and not try to check where they're pointed to. wdyt?

Reply:
I see... what do you think about this alternative solution:

```
  const {router, organization, routerContext} = initialData;

    mountWithTheme(
      <WidgetCard
        api={api}
        organization={organization}
        widget={widget}
        selection={selection}
        isEditing={false}
        onDelete={() => undefined}
        onEdit={() => undefined}
        onDuplicate={() => undefined}
        renderErrorMessage={() => undefined}
        isSorting={false}
        currentWidgetDragging={false}
        showContextMenu
        widgetLimitReached={false}
      />,
      {
        context: routerContext,
      }
    );

    userEvent.click(await screen.findByLabelText('Widget actions'));

    expect(screen.getByText('Duplicate Widget')).toBeInTheDocument();

    userEvent.click(screen.getByRole('menuitemradio', {name: 'Open in Issues'}));

    expect(router.push).toHaveBeenCalledWith(
      '/organizations/org-slug/issues/?query=event.type%3Adefault&sort=freq&statsPeriod=14d'
    );
```

we could click on the menuItem and see what the `router.push` receives

Reply:
oo that works great! updated the tests accordingly